documentation/book/proc-metrics-config.adoc

```diff
@@ -0,0 +1,22 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-setup.adoc
+[id='proc-metrics-kafka-{context}']
+
+= Copying JMX metrics configuration to a Kafka resource
```

Comment:
```suggestion
= Copying Prometheus metrics configuration to a Kafka resource
```

Reply:
Ah, we're going with Prometheus not Kafka. :thumbsup: